lib/eggs/custom_egg.dart

```diff
@@ -0,0 +1,106 @@
+import 'package:parabeac_core/controllers/main_info.dart';
+import 'package:parabeac_core/design_logic/design_node.dart';
+import 'package:parabeac_core/generation/generators/import_generator.dart';
+import 'package:parabeac_core/generation/generators/pb_generator.dart';
+import 'package:parabeac_core/generation/generators/plugins/pb_plugin_node.dart';
+import 'package:parabeac_core/generation/generators/value_objects/file_structure_strategy/commands/write_symbol_command.dart';
+import 'package:parabeac_core/interpret_and_optimize/entities/injected_container.dart';
+import 'package:parabeac_core/interpret_and_optimize/entities/interfaces/pb_injected_intermediate.dart';
+import 'package:parabeac_core/interpret_and_optimize/entities/subclasses/pb_attribute.dart';
+import 'package:parabeac_core/interpret_and_optimize/helpers/pb_context.dart';
+import 'package:parabeac_core/interpret_and_optimize/value_objects/point.dart';
+import 'package:parabeac_core/interpret_and_optimize/entities/subclasses/pb_intermediate_node.dart';
+import 'package:uuid/uuid.dart';
+import 'package:recase/recase.dart';
+
+class CustomEgg extends PBEgg implements PBInjectedIntermediate {
+  @override
+  String semanticName = '<custom>';
+  CustomEgg(
+    Point topLeftCorner,
+    Point bottomRightCorner,
+    String name, {
+    PBContext currentContext,
+  }) : super(topLeftCorner, bottomRightCorner, currentContext, name) {
+    addAttribute(PBAttribute('child'));
+    generator = CustomEggGenerator();
+  }
+
+  @override
+  void addChild(PBIntermediateNode node) {
+    getAttributeNamed('child').attributeNode = node;
+  }
+
+  @override
+  void alignChild() {
+    var tempNode = InjectedContainer(
+      child.bottomRightCorner,
+      child.topLeftCorner,
+      child.name,
+      child.UUID,
+    )..addChild(child);
+
+    getAttributeNamed('title').attributeNode = tempNode;
```

Comment:
I believe the attribute name should be `title` 🤔

Reply:
you are correct

Reply:
Actually, sorry. I looked more closely at the code, I'm not even sure if we should wrap custom eggs inside containers. Do you think we should align custom eggs at all? 🤔